user/running-build-in-debug-mode.md

```diff
@@ -205,3 +205,17 @@ Press `q` to exit the log scroll mode.
 Once you exit from all the live `tmate` windows, the debug VM will terminate
 after resetting the job's status to the original status before you restarted it.
 No more phases (`before_install`, `install`, etc.) will be executed.
+
+## Known issues
+
+### Node.js debug builds need to set up the execution environment
+
+When running a Node.js job in the debug mode, your `node` and `npm`
+commands may not be set up according to the build configuration.
+Running
```

Comment:
I think you can make this more clear by saying

To set up the debug environment  inthe same ways as the Node.js image, run:

```
nvm install $TRAVIS_NODE_VERSION
```

(As an aside, it is better to be certain about these things. :-D

- "should"? but how could I tell?
- "properly" what is a properly set up environment.

Reply:
That sounds clearer and more concise.